website/static/js/folderPickerNodeConfig.js

```diff
@@ -477,6 +480,24 @@ var FolderPickerViewModel = oop.defclass({
                 self.loading(false);
                 // Set flag to prevent repeated requests
                 self.loadedFolders(true);
+            },
+            resolveRows: function(item) {
+                item.css = '';
+                return [
+                {
+                    data : 'name',  // Data field name
+                    folderIcons : true,
+                    filter : false,
+                    custom : function(item, col) {
+                        return m('span', decodeURIComponent(item.data.name));
```

Comment:
The use of `decodeURIComponent` here suggests that we are sending URI-decoded content to the front end. Is that correct? If so...

1. Is this true for all addons? (given that this is a generic shared file not specific to google drive)
2. We do have some places where we uri-decode on the backend. ([for folder names](https://github.com/CenterForOpenScience/osf.io/pull/4261/files#diff-7e1f370b53c76f951cb3f1019a348038R123)). If possible, we shouldn't be decoding in different places piecemeal; there should be a unified standard (data gets all sent decoded, or it doesn't)